release/ray_release/buildkite/step.py

```diff
@@ -76,7 +76,11 @@ def get_step(
             )
         concurrency_limit = CONCURRENY_GROUPS[concurrency_group]
     else:
-        concurrency_group, concurrency_limit = get_concurrency_group(test)
+        if smoke_test:
+            concurrency_test = as_smoke_test(test)
```

Comment:
Should we do it at the upper level: like passing in already smoked test? I'm just worried about other cases where we used the wrong config.

Reply:
With the current structure, we have to do it here. The step is just creating the command to call `run_release_test.sh` with the proper arguments (e.g. `--smoke-test`). The resulting job will then load the test config again and apply the `as_smoke_test`.

Or in other words, we don't actually do anything with the test configuration here.

Reply:
(other than calculating the concurrency groups, and adjusting step labels)